ksql-cli/src/test/java/io/confluent/ksql/CliTest.java

```diff
@@ -168,9 +168,9 @@ private static Properties defaultServerProperties() {
     Map<String, Object> startConfigs = genDefaultConfigMap();
     startConfigs.put("num.stream.threads", 4);
 
-    startConfigs.put(SINK_NUMBER_OF_REPLICATIONS, 1);
-    startConfigs.put(SINK_NUMBER_OF_PARTITIONS, 4);
-    startConfigs.put(SINK_WINDOW_CHANGE_LOG_ADDITIONAL_RETENTION, 1000000);
+    startConfigs.put(SINK_NUMBER_OF_REPLICATIONS_PROPERTY, 1);
```

Comment:
Hey hojjat - why is there a _PROPERTY added - it seems unnecessary? (personal preferance is with the _PROPERTY)